bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
Actually, that's right. I had to resort to something like this: https://github.com/ocsigen/lwt/blob/611ef1833c7aa0ce53221c7589b2673cc17d3063/lwt_ssl.opam#L21 which only "worked" because the constraint syntax was incorrect.